CHANGELOG.md

```diff
@@ -13,6 +13,11 @@ and this project adheres to the versioning scheme outlined in the [README.md](RE
   from the source code are now attached to the `SymbolicExpression` nodes. This
   will be useful for tools that use the Clarity library to analyze and
   manipulate Clarity source code, e.g. a formatter.
+- Added a new RPC endpoint to query the node's health. A node is considered healthy if it
+  is caught up with its bootstrap peers. The endpoint returns 
+  200 if it is synced to its peers' chaintips, 513 if the query was unsuccessful (no viable 
```

Comment:
Looks like this changelog entry is out of sync with the code's current behavior.